internal/controlplane/handlers_authz_test.go

```diff
@@ -232,7 +233,9 @@ func TestProjectAuthorizationInterceptor(t *testing.T) {
 					ID: projectID,
 				},
 			},
-			rpcErr: util.UserVisibleError(codes.PermissionDenied, "user is not authorized to perform this operation"),
+			rpcErr: util.UserVisibleError(
+				codes.PermissionDenied,
+				fmt.Sprintf("user is not authorized to perform this operation on project %q", projectID)),
```

Comment:
At some point we may want to catalogue these?

Reply:
this would be handy!